weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/ObjectViewer.tsx

```diff
@@ -487,16 +491,30 @@ export const ObjectViewer = ({
           const isCustomWeaveType = isCustomWeaveTypePayload(
             params.model.value
           );
-          if (
-            isNonRefString ||
-            (isArray && USE_TABLE_FOR_ARRAYS) ||
-            isTableRef ||
-            isCode ||
-            isCustomWeaveType
-          ) {
-            return 'auto';
+          if (!params.model.isLeaf) {
```

Comment:
it is critical that the row height is static so that virtualization does not cause bouncy results.